be/src/vec/functions/function_json.cpp

```diff
@@ -407,18 +407,36 @@ struct JsonParser<'3'> {
     }
 };
 
+template <>
+struct JsonParser<'4'> {
+    // time
+    static void update_value(StringParser::ParseResult& result, rapidjson::Value& value,
+                             StringRef data, rapidjson::Document::AllocatorType& allocator) {
+        value.SetString(data.data + 1, data.size - 2, allocator);
```

Comment:
some comment? what means `data + 1` and `size - 2`

Reply:
done.